content/projects/index.adoc

```diff
@@ -140,3 +140,17 @@ Jenkins Configuration as Code allows to define this whole configuration as a sim
 link:jcasc[*Learn more*]
 
 ---
+
+== Jenkins Operator
+
+image:/images/projects/jenkins-operator/jenkins-operator-icon.png["Jenkins Operator", role=left]
```

Comment:
FTR the logo is already available as /images/logos/kubernetes-operator/kubernetes-operator.png